src/tram/tram/static/js/analyze.js

```diff
@@ -108,3 +174,9 @@ function renderMappings(sentence_id) {
 
     $("#mapping-container").replaceWith($mappingContainer);
 }
+
+function createCallback(sentence_id, mapping_id, refresh_sentences) {
```

Comment:
Sorry for leading you astray on this one callback business. I forgot that JavaScript closures don't bind values in a loop unless you use `let`. I think changing `var mapping` to `let mapping` and then using arrow syntax would do the trick, but I don't want to beat a dead horse. We can clean up the code in CTID-26.